icaruscode/TPC/ICARUSWireCell/icarus/simparams.jsonnet

```diff
@@ -10,9 +10,9 @@ base {
     // Transverse diffusion constant
     DT :  8.8 * wc.cm2/wc.s,
     // Electron lifetime
-    lifetime : 9.6*wc.ms,
+    lifetime : 3.5*wc.ms,
```

Comment:
This electron liftime should not impact simulation, as it is set with https://github.com/SBNSoftware/icaruscode/blob/c988c8617a4c104e5c446517f9719c8d3bb6fd06/icaruscode/TPC/ICARUSWireCell/wcls-multitpc-sim-drift-simchannel.fcl#L60